tools/m2ctx.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import sys
+import subprocess
+import tempfile
+from pathlib import Path
+
+script_dir = Path(os.path.dirname(os.path.realpath(__file__)))
+root_dir = script_dir / ".."
+src_dir = root_dir / "src"
+
+# Project-specific
+CPP_FLAGS = [
+    "-Iinclude",
+    "-Ilib/ultralib/include",
+
+    "-DBUILD_VERSION=VERSION_J",
+    "-D_FINALROM",
+    "-DF3DEX_GBI_2",
+    "-DGBI_DOWHILE",
+    "-D__sgi",
+    "-D_LANGUAGE_C",
+    "-DNON_MATCHING",
+    "-D_Static_assert(x, y)=",
+    "-D__attribute__(x)="
+    "-D_MIPS_SZLONG=32",
+    "-ffreestanding",
+    "-DM2CTX",
+
+    "-std=gnu89",
+]
+
+# Read through the processes context and replace whatever
+def custom_replacements(output):
```

Comment:
Looks like this function was accidentally carried over from another decomp. It should be removed since it doesn't apply to this project.